server/data/helpers/tests/teacherHelper.spec.js

```diff
@@ -1,39 +1,48 @@
 const db = require('../../../config/dbConfig.js');
 const teacherHelper = require('../teacherHelper.js');
 
-afterEach(async () => {
-	await db('teachers').truncate();
+afterAll(async (done) => {
+	await db.raw('TRUNCATE TABLE teachers RESTART IDENTITY CASCADE').then(() => db.seed.run())
```

Comment:
I think we could remove the truncate statement here since it's been added to the seeds

Reply:
I have to play with this a bit, I remember when I didn't add this to the test files, it would give me the foreign key error in the tests

Reply:
Gotcha :+1: